src/controllers/artist.js

```diff
@@ -0,0 +1,75 @@
+const db = require('../db/index');
+
+const artistController = (req, res) => {
+  res.status(201).json({
+    name: req.body.name,
+    genre: req.body.genre,
+  });
+};
+
+const createArtist = async (req, res) => {
+  const { name, genre } = req.body;
+
+  try {
+    const {
+      rows: [artist],
+    } = await db.query(
+      `INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *`,
+      [name, genre]
+    );
+    res.status(201).json(artist);
+  } catch (err) {
+    res.status(500).json(err.message);
+  }
+};
+
+const getArtists = async (req, res) => {
```

Comment:
parameter req can be replaced with _ as it's not used in the function, makes the code more readable